src/base/static/state/ducks/map-config.d.ts

```diff
@@ -0,0 +1,22 @@
+import { EasingFunction } from "react-map-gl";
```

Comment:
FYI we can start using `.d.ts` files for our ducks, which allows us to define and re-use interfaces without having to port the entire file over to TS.

Previously we were duplicating `InitialMapViewport` definitions. This should fix that.